scripts/update-contributors.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const { spawnSync } = require('child_process')
+
+const filteredNames = ['renovate-bot']
+
+const people = spawnSync('all-contributors', ['check'], {
+  stdio: 'pipe',
+  encoding: 'utf-8',
+})
+
+const peopleList = people.output[1].slice(people.output[1].indexOf('\n', 25)).split(',')
+
+const trimList = peopleList.map(name => {
+  if (name.trim() !== '') {
+    return name.trim()
+  }
+  return null
+})
+
+if (trimList.length > 0) {
+  trimList.map(name => {
```

Comment:
Use a `.forEach` here instead of `.map`. `.map` should only be used if you are using the return value. `.forEach` is for general iterating. 👍

Reply:
Totally brain farted and forgot about `.forEach` on that! Thanks, I'll replace it.